src/test/scala/com/ibm/couchdb/api/DocumentsSpec.scala

```diff
@@ -218,6 +218,35 @@ class DocumentsSpec extends CouchDbSpecification {
       aliceRes2.doc.age mustEqual fixAlice.age + 1
     }
 
+    "Update a document with attachments" >> {
```

Comment:
should this test come after the "Create and get a document with attachments"? just to maintain a logical flow